jaxrs/pom.xml

```diff
@@ -128,6 +139,12 @@
             <version>${yasson.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Just asking: Why do we need geronimo?

Reply:
We need a microprofile config provider to pass the test.
I suppose this is required by the new resteasy ...